source/cipher_openssl.c

```diff
@@ -323,6 +323,7 @@ int aws_cryptosdk_sig_get_pubkey(
 
 int aws_cryptosdk_sig_get_privkey(
     const struct aws_cryptosdk_sig_ctx *ctx, struct aws_allocator *alloc, struct aws_string **priv_key) {
+    AWS_PRECONDITION(ctx != NULL);
```

Comment:
Why not assert `aws_cryptosdk_sig_ctx_is_valid_cbmc()`?

Reply:
Updated!